gril/ril_constants.h

```diff
@@ -447,6 +447,7 @@
 #define RIL_UNSOL_DC_RT_INFO_CHANGED 1041
 #define RIL_UNSOL_RADIO_CAPABILITY 1042
 #define RIL_UNSOL_ON_SS 1043
+#define RIL_UNSOL_ON_SS_LL 11055
```

Comment:
If it's a vendor-specific constant, please put it in a vender-specific file. Sees https://github.com/ubports/ofono/blob/xenial/drivers/qcommsimmodem/qcom_msim_constants.h

Reply:
The constant is unused ATM and I could also not find a real usage in Android/Lineage parts, so I removed it again.